circle.yml

```diff
@@ -31,6 +37,12 @@ test:
     - scripts/merge_conflicts.sh
     # Run htmlproofer.
     - bundle exec rake
+    # @todo, Switch the test to http://"$normalize_branch"-static-docs.pantheon.io
```

Comment:
@rachelwhitton Do you know if/how I can get this variable available in this yml file? Or will the running of this test have to move to a script where the multidev URL can be constructed?